src/libs/NumberUtils.js

```diff
@@ -47,4 +47,18 @@ function generateHexadecimalValue(num) {
         .toUpperCase();
 }
 
-export {rand64, generateHexadecimalValue};
+/**
+ * Generates a random integer between a and b
+ * It's and equivalent of _.random(a, b)
+ *
+ * @param {Number} a
+ * @param {Number} b
+ * @returns {Number} random integer between a and b
+ */
+function generateRandomInt(a, b) {
+    const lower = Math.ceil(Math.min(a, b));
+    const upper = Math.floor(Math.max(a, b));
+    return Math.floor(lower + Math.random() * (upper - lower + 1));
+}
+
```

Comment:
Should it be TS as well?

Reply:
This file hasn't been migrated to TypeScript yet, I am pretty sure there is a separate task for it